### ISSUE TITLE
convert exposecontroller to be an init container to avoid need for admin role

### ISSUE THREAD

**Issue by rawlingsj** [fabric8io/fabric8-online#8]

**Comment by rawlingsj** [comment]
I don't think we need this anymore correct @jstrachan?  On openshift we can avoid the exposecontroller and use Routes?

**Comment by jstrachan** [comment]
we still need exposecontroller for the oauthclient stuff for consoles unfortunately - which still needs admin

**Comment by rawlingsj** [comment]
That's a good point, I guess it's only needed in the SaaS though, so not too bad.

**Comment by jstrachan** [comment]
yeah- we don't need exposecontroller at all on a per user/tenant team though - its only needed on the SaaS to run the consoles
